### ISSUE TITLE
Support for Honeywell wireless sensors? 

### ISSUE THREAD

**Issue by jeffreydwalter** [taconsultingllc/hornet-nest-alarm-panel#3]
Hello, will I be able to connect my existing Honeywell wireless sensors to this system? I have a mix of wired and wireless zones.

**Comment by taconsultingllc** [comment]
No, it's not likely that feature will make it on the roadmap.  There are other products on the market that thou that you should check out if you already haven't.  One specifically is the EnvisaLink which is designed to integrate with DCS and Honeywell systems and can integrate those existing systems into Home Assistant.

EnvisaLink: https://www.eyezon.com/evl4.html

Home Assistant Integration: https://www.home-assistant.io/integrations/envisalink/
